MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -167,9 +167,13 @@ public class NavigationDirections: Directions {
                         return completionHandler(nil, nil, error as NSError)
                     }
                 } else {
-                    
                     DispatchQueue.main.async {
                         let response = options.response(from: json)
+                        let responseEndDate = Date()
+                        for route in response.1 ?? [] {
```

Comment:
This is just to demonstrate how we’d go about collecting analogous statistics about client-side route calculation if that were to become relevant in the future.